.github/workflows/build-and-deploy-notifications.yml

```diff
@@ -0,0 +1,64 @@
+name: Build and Deploy
+
+on:
+  push:
+    branches:
+    - main
+    - deploy-resources
+  pull_request:
+    branches:
+    - main
+
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: Build & test
```

Comment:
There's quite a lot missing from the build job, can you copy over everything from the version of build.yml that's currently in the `main` branch

Reply:
removed